### PR TITLE
[ADD] New module Sale Order Invoice Link

### PR DESCRIPTION
**Pull request by mileo**

### CODE REVIEW

**Comment by mileo** [comment]
cc @pedrobaeza  https://github.com/OCA/sale-workflow/pull/352

File: sale_order_invoice_link/models/account_invoice.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import api, fields, models
+
+
+class AccountInvoice(models.Model):
+    _inherit = 'account.invoice'
+
+    sale_ids = fields.Many2many(
+        string='Sale Orders2',
```
Comment (pedrobaeza) [inline sale_order_invoice_link/models/account_invoice.py]: s/Orders2/Orders

File: sale_order_invoice_link/models/account_invoice.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        comodel_name='sale.order',
+        compute="_compute_sale_ids"
+    )
+
+    @api.multi
+    def _compute_sale_ids(self):
+        for invoice in self:
+            invoice.sale_ids = self.env['sale.order']
+            for line in invoice.invoice_line_ids:
```
Comment (pedrobaeza) [inline sale_order_invoice_link/models/account_invoice.py]: Do directly:

```
invoice.sale_ids = invoice.mapped('sale_line_ids.order_id')
```

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Maybe you can put an smart button for accessing linked sales orders in "full screen" instead of a tab

**Pull request closed without merging**